src/dmd/func.d

```diff
@@ -167,7 +196,13 @@ extern (C++) class FuncDeclaration : Declaration
 
     const(char)* mangleString;          /// mangled symbol created from mangleExact()
 
-    Identifier outId;                   /// identifier for out statement
+    @property Identifier outId(){       /// identifier for result
```

Comment:
This would be dead code now, so remove?

Reply:
It is not dead code, but I can turn it into a more appropriately-named `bool` function.